src/main/java/org/prebid/server/cache/CacheService.java

```diff
@@ -162,13 +156,12 @@ public String getEndpointPath() {
                 }
             }
 
-            result = CompositeFuture.all(
+            result = doCacheOpenrtb(
                     getCacheBids(cacheContext.isShouldCacheBids(), bids, impIdToTtl, impWithNoExpExists,
```

Comment:
Please extract `getCacheBids(...)` result to separate variable. Make the same for `getVideoCacheBids(...)`.